.gitlab-ci.yml

```diff
@@ -343,33 +351,37 @@ build:docker-multiplatform:
   after_script:
     - cd $CI_PROJECT_DIR
     - mkdir logs && cp /root/.npm/_logs/* logs/
-  only:
-    - staging
-  when: delayed
-  start_in: 15 minutes
+  # only:
+  #   - staging
   artifacts:
     expire_in: 2w
     paths:
       - tests/e2e_tests/test-results
       - logs
     when: always
   tags:
-    - mender-qa-worker-generic-light
+    - hetzner-amd-beefy-privileged
```

Comment:
Oh ok, would it be ok to keep the staging tests on Hetzner (since they would only run on the staging branch) and just change the regular e2e tests?

Reply:
> Oh ok, would it be ok to keep the staging tests on Hetzner (since they would only run on the staging branch) and just change the regular e2e tests? The main reason is basically the webkit flavour of the tests requiring more power than light has and if we can run them all on Hetzner that wouldn't need an override 🙃.

That would be great!